client/src/components/Navigation.js

```diff
@@ -88,6 +88,9 @@ class Navigation extends Component {
               <NavItem>
                 <NavLink href="/settings">Settings</NavLink>
```

Comment:
Should this use the `tag=` and `to=`? the `master` branch has this on line 106 (master branch).